### PR TITLE
[Method References 2] LPS-164454 Convert method reference to ServiceTrackerMap in module dynamic-data-mapping-api

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I convert method reference to ServiceTrackerMap in module dynamic-data-mapping-api.

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results ? Thanks!

**Comment by Kvale1733** [comment]
@W2jx No regressions.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMDisplayRegistry.java [hunk 1]
```diff
@@ -45,34 +45,35 @@ public String[] getPortletIds() {
...
-	protected void unsetDDMDisplay(DDMDisplay ddmDisplay) {
-		_ddmDisplays.remove(ddmDisplay.getPortletId());
+	@Deactivate
+	protected void deactivate() {
+		_serviceTrackerMap.close();
 	}
 
 	private DDMDisplay _getDDMDisplay(String portletId) {
-		return _ddmDisplays.get(portletId);
+		return _serviceTrackerMap.getService(portletId);
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMDisplayRegistry.java:64]: @W2jx Please add an SF commit to inline these private methods, as they seem to be only used by their public counterparts.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMDisplayRegistry.java (hunk 1 above)
Comment (W2jx) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMDisplayRegistry.java:64]: I will fix it.

**Comment by dantewang** [comment]
@W2jx In order to avoid too many open PRs hanging here, I'm going to close this PR instead of waiting for a force push. Please resend, thanks!

**Pull request closed without merging**